seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/config/FakeConfig.java

```diff
@@ -90,6 +205,174 @@ public static FakeConfig buildWithConfig(Config config) {
             }
             builder.fakeRows(rows);
         }
+        if (config.hasPath(STRING_TEMPLATE.key())) {
+            builder.stringTemplate(config.getStringList(STRING_TEMPLATE.key()));
+        }
+        if (config.hasPath(TINYINT_TEMPLATE.key())) {
+            builder.tinyintTemplate(config.getIntList(TINYINT_TEMPLATE.key()));
+        }
+        if (config.hasPath(SMALLINT_TEMPLATE.key())) {
+            builder.smallintTemplate(config.getIntList(SMALLINT_TEMPLATE.key()));
+        }
+        if (config.hasPath(INT_TEMPLATE.key())) {
+            builder.intTemplate(config.getIntList(INT_TEMPLATE.key()));
+        }
+        if (config.hasPath(BIGINT_TEMPLATE.key())) {
+            builder.bigTemplate(config.getLongList(BIGINT_TEMPLATE.key()));
+        }
+        if (config.hasPath(FLOAT_TEMPLATE.key())) {
+            builder.floatTemplate(config.getDoubleList(FLOAT_TEMPLATE.key()));
+        }
+        if (config.hasPath(DOUBLE_TEMPLATE.key())) {
+            builder.doubleTemplate(config.getDoubleList(DOUBLE_TEMPLATE.key()));
+        }
+        if (config.hasPath(DATE_YEAR_TEMPLATE.key())) {
+            builder.dateYearTemplate(config.getIntList(DATE_YEAR_TEMPLATE.key()));
+        }
+        if (config.hasPath(DATE_MONTH_TEMPLATE.key())) {
+            builder.dateMonthTemplate(config.getIntList(DATE_MONTH_TEMPLATE.key()));
+        }
+        if (config.hasPath(DATE_DAY_TEMPLATE.key())) {
+            builder.dateDayTemplate(config.getIntList(DATE_DAY_TEMPLATE.key()));
+        }
+        if (config.hasPath(TIME_HOUR_TEMPLATE.key())) {
+            builder.timeHourTemplate(config.getIntList(TIME_HOUR_TEMPLATE.key()));
+        }
+        if (config.hasPath(TIME_MINUTE_TEMPLATE.key())) {
+            builder.timeMinuteTemplate(config.getIntList(TIME_MINUTE_TEMPLATE.key()));
+        }
+        if (config.hasPath(TIME_SECOND_TEMPLATE.key())) {
+            builder.timeSecondTemplate(config.getIntList(TIME_SECOND_TEMPLATE.key()));
+        }
+        if (config.hasPath(TINYINT_MIN.key())) {
+            int tinyintMin = config.getInt(TINYINT_MIN.key());
+            if (tinyintMin < TINYINT_MIN.defaultValue() || tinyintMin > TINYINT_MAX.defaultValue()) {
+                throw new FakeConnectorException(CommonErrorCode.ILLEGAL_ARGUMENT,
+                        TINYINT_MIN.key() + " should >= " + TINYINT_MIN.defaultValue() +
+                                " and <= " + TINYINT_MAX.defaultValue());
```

Comment:
Using String.format may be better.
There's a lot of code duplication like this, how about wrapping it as a method to return an error message?